examples/graphene_apps/python_worker/avalon_worker/tests/test_send_work_order.py

```diff
@@ -0,0 +1,178 @@
+                                                session_key_iv,
+                                                worker_signup_json)
+    json_request = _create_json_request("ProcessWorkOrder",
+                                        input_json_str)
+    # Send work order
+    context = zmq.Context()
+    logger.info("Send work order request\n")
+    response = _send_request_zmq(context, zmq_url, json.dumps(json_request))
+    # Parse response
+    response_json = json.loads(response)
+    # Verify response signature
+    sign = worker_signing.WorkerSign()
+    worker_verify_key = worker_signup_json["verifying_key"].encode("UTF-8")
+    verify = sign.verify_response_signature(response_json, worker_verify_key)
+    if verify:
+        logger.info("Work order result verification success")
+    else:
+        logger.error("Work order result verification fail")
+    # Get work order result
+    if "result" in response_json:
+        decrypted_out_message = _get_work_order_result(response_json,
+                                                       session_key,
+                                                       session_key_iv)
+        logger.info("Work order result : {}".format(decrypted_out_message))
+    elif "error" in response_json:
+        logger.error("Work order execution failed")
+
+
+def _send_request_zmq(context, zmq_url, data):
+    try:
+        socket = context.socket(zmq.REQ)
+        socket.connect(zmq_url)
+        socket.send_string(data, flags=0, encoding='utf-8')
+        replymessage = socket.recv()
+        replymessage = replymessage.decode("UTF-8")
+        socket.disconnect(zmq_url)
+    except Exception as ex:
+        replymessage = "Error while processing work order"
+        logger.error("Exception: {} args {} details {}"
+                     .format(type(ex), ex.args, ex))
+    return replymessage
+
+
+def _create_json_request(method_name, params):
+    json_request = {}
+    json_request["jsonrpc"] = "2.0"
+    json_request["id"] = random.randint(0, 100000)
+    json_request["method"] = method_name
+    json_request["params"] = params
+    return json_request
+
+
+def _generate_session_key_iv():
+    encrypt = worker_encryption.WorkerEncrypt()
+    session_key = encrypt.generate_session_key()
+    session_key_iv = encrypt.generate_iv()
+    return (session_key, session_key_iv)
+
+
+def _create_work_order_request(workload_id, input_data_str,
+                               session_key, session_key_iv,
+                               worker_signup_json):
+    worker_enc_key_str = worker_signup_json["encryption_key"]
+
+    encrypt = worker_encryption.WorkerEncrypt()
+    encrypted_session_key = encrypt.encrypt_session_key(
+        session_key, worker_enc_key_str)
+
+    msg = input_data_str
+    msg_bytes = msg.encode("UTF-8")
+    encrypt_msg_bytes = encrypt.encrypt_data(msg_bytes,
+                                             session_key, session_key_iv)
+    input_json = dict()
```

Comment:
Can we use workOrderParams from sdk to create the work order request? Reuse the code.

Reply:
same as https://github.com/hyperledger/avalon/pull/612/files#r451024639